src/workflow/workflow.c

```diff
@@ -52,23 +63,8 @@ static void _select_orientation_done(bool upside_down)
     ui_screen_stack_switch(show_logo);
 }
 
-void workflow_start(void)
-{
-    usb_start(hww_setup);
-    ui_screen_stack_pop_all();
-    ui_screen_stack_push(info_centered_create("See the BitBox App", NULL));
-}
-
-void workflow_change_state(workflow_state_t state)
+void start_orientation_screen(void)
```

Comment:
prefix with `workflow_` based on the filename. We generally do this as a poor man's namespacing ;) and it helps to know roughly in what area a function lives when reading code.

Reply:
Amended e35498b24bf0ba762babf224fdc8e08edd2d4e